pkg/sqlmigrations/migrations.go

```diff
@@ -1020,9 +1020,27 @@ func migrateMutationJobForTable(
 			continue
 		}
 		log.VEventf(
-			ctx, 2, "job %d: found corresponding mutation %d on table %d",
+			ctx, 2, "job %d: found corresponding MutationJob %d on table %d",
 			*job.ID(), mutationJob.MutationID, tableDesc.ID,
 		)
+		var mutation *sqlbase.DescriptorMutation
+		found := false
+		for i := range tableDesc.Mutations {
+			if tableDesc.Mutations[i].MutationID == mutationJob.MutationID {
+				mutation = &tableDesc.Mutations[i]
+				found = true
+				break
+			}
+		}
+		if !found {
+			// In theory, MutationJobs and Mutations are synced with each other, so
```

Comment:
I think it may be worth mentioning what is meant by "synced" here. Additionally, it may also be worth clarifying that `MutationJobs[i]` corresponds to `Mutations[i]` on 19.2 - which is the version we're migrating from (I don't think this is true anymore in 20.1 due to primary schema changes).